go/teams/service_helper.go

```diff
@@ -175,6 +175,14 @@ func RemoveMember(ctx context.Context, g *libkb.GlobalContext, teamname, usernam
 	return t.ChangeMembership(ctx, req)
 }
 
+func LeaveTeam(ctx context.Context, g *libkb.GlobalContext, teamname string, permanence bool) error {
+	t, err := Get(ctx, g, teamname)
```

Comment:
This seems to have changed on master - I don't think I need to specify any refreshers or a particular keyGeneration for `get.go:GetForApplicationByStringName`, is this correct?